### PR TITLE
Merge #11648: [tests] Add messages.py 

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: test/functional/test_framework/mininode.py [hunk 1]
```diff
@@ -35,18 +35,45 @@
...
-mininode_socket_map = dict()
-
-# One lock for synchronizing all data access between the networking thread (see
-# NetworkThread below) and the thread running the test logic.  For simplicity,
-# NodeConn acquires this lock whenever delivering a message to a NodeConnCB,
-# and whenever adding anything to the send buffer (in send_message()).  This
-# lock should be acquired in the thread running the test logic to synchronize
-# access to any data shared with the NodeConnCB or NodeConn.
-mininode_lock = threading.RLock()
+MESSAGEMAP = {
```
Comment (UdjinM6) [inline test/functional/test_framework/mininode.py]: nit: this was re-ordered alphabetically in the original PR, should probably do the same to minimize potential merge conflicts in the future

File: test/functional/test_framework/messages.py [hunk 2]
```diff
@@ -0,0 +1,1437 @@
+
```
Comment (UdjinM6) [inline test/functional/test_framework/messages.py]: Should not have an empty line here

**Comment by PastaPastaPasta** [comment]
force pushed rebasing on develop and fixing conflicts caused by adding devnets to mininode

File: test/functional/test_framework/mininode.py (hunk 1 above)
Comment (PastaPastaPasta) [reply test/functional/test_framework/mininode.py]: https://github.com/dashpay/dash/pull/3372/commits/f7ae32703ae0e80f85f63945ed99914eff8cb65f

File: test/functional/test_framework/messages.py (hunk 2 above)
Comment (PastaPastaPasta) [reply test/functional/test_framework/messages.py]: https://github.com/dashpay/dash/pull/3372/commits/cb91d15055edc530213ca75cb145fd9220c10e82

**Comment by UdjinM6** [comment]
Looks like some messages are/were missing fb77e3a634

**Comment by PastaPastaPasta** [comment]
done

**Pull request merged**